src/ratelimit.ts

```diff
@@ -127,11 +129,11 @@ export abstract class Ratelimit<TContext extends Context> {
    *  return "Yes"
    * ```
    */
```

Comment:
Maybe a comment like this:

@param req.rate - The rate at which tokens will be added or consumed from the token bucket. A higher rate allows for more requests to be processed. Defaults to 1 token per interval if not specified.

Reply:
Yes I will put that